src/screens/SettingsScreen.js

```diff
@@ -240,6 +240,7 @@ export const SettingsScreen = () => {
         />
       )}
       {selectedFilterId === TOP_FILTER.LIST_TYPES && <ListSettings />}
+      {/* TODO: this will crash now as the props were changed in `AugmentedReality`, right? */}
       {selectedFilterId === TOP_FILTER.AR_DOWNLOAD_LIST && (
         <AugmentedReality onSettingsScreen tourID="579" />
       )}
```

Comment:
this will crash now as the props were changed in `AugmentedReality`, right?

Reply:
There will be no application crash here. Only the list is not loaded. But I think I solved this problem in [984398](https://github.com/ikuseiGmbH/smart-village-app-app/pull/608/commits/a2eab2d01e81cd949d8827aeb854f772ad59f304).